src/internal/packager/helm/images.go

```diff
@@ -0,0 +1,118 @@
+package helm
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/defenseunicorns/zarf/src/types"
+	"github.com/goccy/go-yaml"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	kustypes "sigs.k8s.io/kustomize/api/types"
+)
+
+// FindFluxImages pulls the raw file from the https repo hosting bigbang
+// Will not work for private/offline/nongitlab based hostings
+func FindFluxImages(bigbangrepo, version string) ([]string, error) {
+	images := make([]string, 0)
+	bigbangrepo = strings.TrimSuffix(bigbangrepo, ".git")
+	rawFile := fmt.Sprintf("%s/-/raw/%s/base/flux/kustomization.yaml", bigbangrepo, version)
```

Comment:
Instead of doing this, can we download repo to temp like we do for Helm charts? https://github.com/defenseunicorns/zarf/blob/42250fda110492ca7a366f8000f12ff06f973d22/src/internal/packager/git/pull.go#L21